src/main/java/org/mifos/pheeBillPay/camel/routes/BillerFetchRouteBuilder.java

```diff
@@ -32,28 +35,55 @@ public void configure() {
             logger.debug("Bill Id: " + billId);
```

Comment:
Dont use + for concatenation use {} instead. This practice will make the code null safe